### PR TITLE
SISRP-23764 - Academics > Advising > Appointment Self Service - Return to CalCentral link is not working

### PR DESCRIPTION
**Pull request by dfkaye**
https://jira.berkeley.edu/browse/SISRP-23764

### CODE REVIEW

File: app/models/campus_solutions/link.rb [hunk 1]
```diff
@@ -68,6 +68,7 @@ def get_url(url_id, placeholders = {})
           link[:name] = link.delete(:description)
           link[:title] = link.delete(:comments)
           if !link[:showNewWindow]
+            link[:isCsLink] = true
             link['IS_CS_LINK'] = true
```
Comment (raydavis) [inline app/models/campus_solutions/link.rb]: Is the "IS_CS_LINK" property used by any higher-level code, @dfkaye ? I don't see it mentioned anywhere as a raw string.

Comment (dfkaye) [inline app/models/campus_solutions/link.rb]: @raydavis Yes - the link collection that gets built in app/models/campus_solutions/advising_resources.rb apparently camelCases it

Comment (raydavis) [inline app/models/campus_solutions/link.rb]: Sorry to bug you again, @dfkaye , but if it camelCased "IS_CS_LINK", wouldn't that have the same effect as if the "IS_CS_LINK" was replaced by "isCsLink"? I guess I'm just wondering, if you took "IS_CS_LINK" out, would you end up with the same payload going to the browser?

Comment (dfkaye) [inline app/models/campus_solutions/link.rb]: @raydavis I tried that - it turned `:isCsLink` into `iscslink`

Comment (raydavis) [inline app/models/campus_solutions/link.rb]: Thanks, @dfkaye . Pretty confusing situation -- we should probably try to clear this redundancy out of the pipes at some point, so as not to have developers copy-and-pasting "isCsLink" and "IS_CS_LINK" combos all over the codebase...

Comment (paulkerschen) [inline app/models/campus_solutions/link.rb]: +1 @raydavis - I for one would support a convention that symbolizes-and-camelizes string keys in these feeds, but leaves symbol keys alone.

**Pull request merged**